src/core/utils/qfieldcloudutils.h

```diff
@@ -99,6 +99,22 @@ class QFieldCloudUtils : public QObject
      */
     Q_INVOKABLE static const QString getProjectId( const QString &fileName );
 
+    /**
+     * Returns user-friendly message.
+     *
+     * @param errorMessage reveiced error message
+     * @return const QString
+     */
+    Q_INVOKABLE static const QString userFriendlyErrorString( const QString &errorMessage );
```

Comment:
Is there a reason to make the return type const? In my experience this doesn't have much effect, so it's not done throughout the QGIS (and Qt) codebases.

Reply:
I don't have a specific reason for making the return type `const`, I was just following the formatting of other functions. However, I feel more safe by using `const`.